# int32 request_type
int32 DETECT_PROGRAM = 1
int32 SYSTEM_REQUEST = 2

# int32 state
int32 SPEAKING = 90
int32 LISTENING = 91
int32 PROCESSING = 92
int32 PREEMTED = 93
int32 RECEIVED = 94
